options.go

```diff
@@ -0,0 +1,80 @@
+package retry
+
+import (
+	"time"
+)
+
+type config struct {
+	tries           uint
+	delay           time.Duration
+	units           time.Duration
```

Comment:
Ty units mi prijdou jako komplikace, ted jsem 5 minut premyslel, jak se pocita ta doba, co to bude spat. Klidne bych tam nechal jen ten delay. Chapu to spravne, ze kdyz dam delay 1s a unit 1h, tak to bude spat hodinu?

Reply:
to je koncept prebrany z `Try::Tiny::Retry` hodi se pro testovani, jinak to neni potreba nastavovat...

Reply:
obecne nemusis koukat na `config` to stejne nikdo neuvidi - to je interni struktura